tests/QueryTest.php

```diff
@@ -824,6 +824,14 @@ public function testWhereBasic()
         );
     }
 
+    public function testWhereExpression()
```

Comment:
Please add functional test (test testing the issue with real DB)

Reply:
If SQL is rendering correctly I see no point in testing on real DB.
I updated the test a bit with more conditions

Reply:
there is - real DB tests a behaviour, rendered conditions can be very easily replaced when modifying something like now